src/hotspot/cpu/aarch64/stubGenerator_aarch64.cpp

```diff
@@ -4891,80 +4882,89 @@ class StubGenerator: public StubCodeGenerator {
       __ bind(LARGE_LOOP_PREFETCH);
         __ prfm(Address(str1, SoftwarePrefetchHintDistance));
         __ prfm(Address(str2, SoftwarePrefetchHintDistance));
-        compare_string_16_bytes_same(DIFF, DIFF2);
-        compare_string_16_bytes_same(DIFF, DIFF2);
+
+        __ align(OptoLoopAlignment);
+        for (int i = 0; i < 4; i++) {
+          __ ldp(tmp1, tmp1h, Address(str1, i * 16));
+          __ ldp(tmp2, tmp2h, Address(str2, i * 16));
+          __ cmp(tmp1, tmp2);
+          __ ccmp(tmp1h, tmp2h, 0, Assembler::EQ);
+          __ br(__ NE, DIFF);
```

Comment:
The line above uses `Assembler::EQ` for the condition code but this line uses `__ NE`. Better to be consistent and use `Assembler::` everywhere.

Reply:
Thanks, I'll fix it.